projects/media-viewer/src/lib/shared/viewer.events.service.ts

```diff
@@ -0,0 +1,25 @@
+import {Injectable} from '@angular/core';
+import {Subject} from 'rxjs';
+
+
+// Viewer Custom-Event Types
+export interface TextSelectionRectangle {
+  x: number;
+  y: number;
+  height: number;
+  width: number;
+}
+export type TextSelected = TextSelectionRectangle[];
+
+
+@Injectable({providedIn: 'root'})
+export class ViewerEventsService {
```

Comment:
Same as above re filename. I'm not really fussed though

Reply:
agreed, but will revise

Reply:
renamed. @linusnorton can you recheck?